lib/restbase.js

```diff
@@ -325,13 +325,20 @@ RESTBase.prototype._request = function(req) {
         });
 
         if (accessRestricted) {
-            return P.all([
-                reqHandlerPromise,
-                childRESTBase._authService.checkPermissions(childRESTBase, childReq)
-            ])
-            .then(function(res) {
-                return res[0];
-            });
+            if (req.method === 'get' || req.method === 'head') {
```

Comment:
Why limit `GET`s as well? `POST`s are a real problem here. Say that an internal service `POST`s something and gets back the hash for it. Then, external clients should be able to retrieve it, shouldn't they?